ion-schema/src/isl/version_based_isl/constraints/open_content.rs

```diff
@@ -0,0 +1,27 @@
+use crate::isl::version_based_isl::IslVersionTrait;
+use ion_rs::element::Element;
+use std::marker::PhantomData;
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct OpenContent<V: IslVersionTrait> {
```

Comment:
question—does `OpenContent` really need to be versioned?

Reply:
No, its not required to be versioned.